src/crepe/facade/SDLContext.h

```diff
@@ -139,6 +139,26 @@ class SDLContext {
 	 */
 	void camera(const Camera & camera);
```

Comment:
het set de camera toch niet?
Het veranderd de viewport?
Doxygen comment kloppen niet en de functie naam is niet zo duidelijk